website/docs/validating-admission-policy.md

```diff
@@ -176,3 +176,26 @@ spec:
 ```
 
 To see this in action, checkout this [demo](https://github.com/open-policy-agent/gatekeeper/tree/master/demo/scoped-enforcement-actions)
+
+## FAQs
+
+<details>
+
+<summary>Do all engines in a ConstraintTemplate are evaluated? Is there a fallback among engines?</summary>
+Only one engine is evaluated for each ConstraintTemplate. K8sNativeValidation engine hold higher priority than Rego engine. There is no fallback mechanism between engines, hence a logical/syntactical error in policy logic is treated as violation depending on the enforcement action set of the Constraints created for ConstraintTemplate.
+
+</details>
+
+<details>
+
+<summary>If I have a template with Rego and CEL, which policy logic will be used in evaluating resources?</summary>
+
+K8sNativeValidation engine hold higher priority than Rego engine, so with a ConstraintTemplate that has both Rego and CEL. Policy logic written in CEL will get evaluated.
+</details>
+
+<details>
+<summary>Can I change the priority of engines per ConstraintTemplate?
+</summary>
+Engine priority cannot be set per ConstraintTemplate.
```

Comment:
```suggestion
No, engine priority cannot be modified.
```

Removing "per ConstraintTemplate" since it makes it look like it can be done but just not per CT.

Reply:
Makes sense